README.md

```diff
@@ -1,39 +1,45 @@
 # fablicop
 
 fablicop is a RuboCop configration gem.
+It assumes your project is using Ruby on Rails, so some Cops prefixed with `Rails` are enabled out of the box.
+In other words, it's not appropriate to use it with non-Rails projects.
```

Comment:
These sentences might be important because the fablicop user cannot use it without Rails cops.

Reply:
That's a good description!